From ddd6a1ad574c18e33804be4122ea08ab6717eae6 Mon Sep 17 00:00:00 2001
From: Vasanthakumar Pandurangan <vasanth@codeaurora.org>
Date: Tue, 23 Oct 2012 12:16:53 +0530
Subject: [PATCH] For Modem ota update in Nand devices, bootloader should
 write a magic cookie in FOTA partition which it already does
 but without ECC. This change is to do the same with ECC

Change-Id: Iadaf6fd2c5197abc618aa249b88b2c4529257eee
Signed-off-by: Vasanthakumar Pandurangan <vasanth@codeaurora.org>
---
 platform/msm_shared/nand.c |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/platform/msm_shared/nand.c b/platform/msm_shared/nand.c
index 08b68dd..b59df28 100644
--- a/platform/msm_shared/nand.c
+++ b/platform/msm_shared/nand.c
@@ -159,13 +159,8 @@ static struct flash_identification supported_flash[] = {
 
 static void set_nand_configuration(char type)
 {
-	if (type == TYPE_MODEM_PARTITION) {
-		CFG0 = CFG0_M;
-		CFG1 = CFG1_M;
-	} else {
 		CFG0 = CFG0_A;
 		CFG1 = CFG1_A;
-	}
 }
 
 static void flash_nand_read_id(dmov_s * cmdlist, unsigned *ptrlist)
-- 
1.7.8.3

